packages/compass-assistant/src/compass-assistant-provider.spec.tsx

```diff
@@ -626,8 +631,9 @@ describe('CompassAssistantProvider', function () {
         userEvent.click(clearButton);
 
         await waitFor(() => {
-          expect(screen.getByTestId('assistant-confirm-clear-chat-modal')).to
-            .exist;
+          expect(
+            screen.getByTestId('assistant-confirm-clear-chat-modal').firstChild
```

Comment:
Using `.firstChild` here because the modal root (`dialog` element) always exists, regardless of it being open or not.